### DISCUSSION TITLE
Using non-text components as Combobox options

### DISCUSSION THREAD

**Discussion by SaavanNanavati** [ariakit/ariakit#1934] [Questions] [answered]
Hi, 

My combobox option is an image alongside text, however only the text renders. It seems like all non-text elements are getting filtered out. Is this expected? How do I fix it?

For reference, I am also using ComboboxMultiselect.

Thanks!

**Comment by SaavanNanavati** [comment, marked as answer]
Ahh I see. The example renders only the value, not props.children. Looks like that fixes it!

**Reply by SaavanNanavati** [reply]
If I use a component as a child, how do I set the aria-label value of the Combobox option? Is that already done through the value prop, or if not what would the workaround be?

**Reply by diegohaz** [reply]
You can pass `aria-label` to the item component if you want. But the browser and assistive technologies will infer the label of the items based on their text content in the DOM. If you're rendering images, it'll use the `alt` text, etc.
